pom.xml

```diff
@@ -357,6 +358,12 @@
                 <version>${bouncycastle.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
+                <groupId>org.apache.commons</groupId>
+                <artifactId>commons-lang3</artifactId>
+                <version>${apache.commons.version}</version>
+                <scope>compile</scope>
```

Comment:
We've got inconsistency creeping on about how we declare scope.   When do we declare scope in the DM, when do we not. I don't have a suggestion about what the rule should be right now.  Ideas anyone?